spec/spec_helper.rb

```diff
@@ -6,6 +6,8 @@
 require 'rspec'
 require 'rspec/autorun' # somehow makes rcov work with rspec
 require 'twitter_cldr'
+require 'coveralls'
+Coveralls.wear!
```

Comment:
I'm not sure how it works, but coveralls already has [a few reports](https://coveralls.io/r/twitter/twitter-cldr-rb) for this repo.